tools/internal_ci/linux/mattkwong_experiment.cfg

```diff
@@ -0,0 +1,25 @@
+# Copyright 2017 gRPC authors.
```

Comment:
you probably didn't intend to merge this file.

Reply:
I needed this to test using a different VM image. Deleted